scripts/package/truenas/truenas.config

```diff
@@ -0,0 +1,66 @@
+#
+# General setup
+#
+CONFIG_LOCALVERSION="+truenas"
+CONFIG_TRUENAS=y
+# end of General setup
+
+#
+# Processor type and features
+#
+CONFIG_FORCE_MAX_ZONEORDER=14
+# end of Processor type and features
+
+# CONFIG_MODULE_SIG_ALL is not set
+
+#
+# Memory Management options
+#
+CONFIG_CMA=y
+# CONFIG_CMA_DEBUG is not set
+# CONFIG_CMA_DEBUGFS is not set
+CONFIG_CMA_AREAS=19
+CONFIG_CMA_SIZE_MBYTES=256
+CONFIG_CMA_ALIGNMENT=12
+# end of Memory Management options
+
+#
+# Device Drivers
+#
+
+#
+# Generic fallback / legacy drivers
+#
+CONFIG_NTB_NETDEV=m
+
+#
+# SCSI device support
+#
+# CONFIG_SCSI_CHELSIO_FCOE is not set
+# end of SCSI device support
+
+# CONFIG_CHELSIO_T4_FCOE is not set
```

Comment:
Does this result in CONFIG_CHELSIO_T4_FCOE being unset in the final merged config, even though it is set in debian_amd64.config?

Reply:
Yes, that is true.